app/services/usps_in_person_proofing/enrollment_helper.rb

```diff
@@ -98,6 +98,21 @@ def handle_standard_error(err, enrollment)
       def analytics(user: AnonymousUser.new)
         Analytics.new(user: user, request: nil, session: {}, sp: nil)
       end
+
+      def transliterate(value)
+        return value unless IdentityConfig.store.usps_ipp_transliteration_enabled
+
+        result = transliterator.transliterate(value)
+        if result.unsupported_chars.size > 0
```

Comment:
I'm not sure the name makes sense to me, but we can probably use `.present?` instead of checking for a size
```suggestion
        if result.unsupported_chars.present?
```